src/single-spa-examples.js

```diff
@@ -10,11 +10,46 @@ singleSpa.declareChildApplication('svelte', () => SystemJS.import('/build/svelte
 singleSpa.declareChildApplication('preact', () => SystemJS.import('/build/preact.app.js'), hashPrefix('/preact'));
 singleSpa.declareChildApplication('iframe-vanilla-js', () => SystemJS.import('/build/vanilla.app.js'), hashPrefix('/vanilla'));
 singleSpa.declareChildApplication('inferno', () => SystemJS.import('/build/inferno.app.js'), hashPrefix('/inferno'));
+singleSpa.declareChildApplication('ember', () => loadEmberScript('/build/ember-app/assets/ember-app.js'), hashPrefix('/ember'));
 
 singleSpa.start();
 
 function hashPrefix(prefix) {
-	return function() {
-		return window.location.hash.indexOf(`#${prefix}`) === 0;
+	return function(location) {
+		return location.hash.indexOf(`#${prefix}`) === 0;
 	}
 }
+
+function loadEmberScript(src) {
```

Comment:
This function is doing way more than it should. Ideally it would just call `SystemJS.import()`, but because of the issues loading ember bundles with systemjs (see https://stackoverflow.com/questions/45438094/issue-in-loading-ember-app-as-child-application-in-single-spa and #33), I'm just implementing a workaround.